nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/summary/ui/process-group-status-listing/process-group-status-table/process-group-status-table.component.ts

```diff
@@ -83,7 +83,14 @@ export class ProcessGroupStatusTable extends ComponentStatusTable<ProcessGroupSt
     @Input() rootProcessGroup!: ProcessGroupStatusSnapshot;
 
     override filterPredicate(data: ProcessGroupStatusSnapshotEntity, filter: string): boolean {
-        const { filterTerm, filterColumn } = JSON.parse(filter);
+        const { filterTerm, filterColumn, filterVersionedFlowState } = JSON.parse(filter);
+        const matchOnVersionedFlowState: boolean = filterVersionedFlowState !== 'All';
+
+        if (matchOnVersionedFlowState) {
+            if (this.formatVersionedFlowState(data) !== filterVersionedFlowState) {
+                return false;
+            }
```

Comment:
If the value of the selection is the enum value, then we shouldn't have to coerce the data into the label displayed to compare when filtering...

```suggestion
            if (data.processGroupStatusSnapshot.versionedFlowState !== filterVersionedFlowState) {
                return false;
            }
```

Reply:
Updated